api/apps/api/src/modules/legacy-project-import/domain/legacy-project-import-checker/legacy-project-import-checker.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Either } from 'fp-ts/Either';
+
+export const legacyProjectImportDoesntExist = Symbol(
+  `doesn't exist legacy project import`,
+);
+export type LegacyProjectImportDoesntExist = typeof legacyProjectImportDoesntExist;
+
+export abstract class LegacyProjectImportChecker {
+  abstract hasImportedLegacyProjectImport(
```

Comment:
sorry about the picky note, but I think the name is unclear - from what the implementations do I think this could be renamed to something like this?

```suggestion
  abstract isLegacyProjectImportCompletedFor(
```

Reply:
(if you feel it should be amended for clarity, please remember to update the function name in test descriptions too)